lib/zooniverse_user.rb

```diff
@@ -41,7 +41,7 @@ def password=(plain_password)
 
   def import
     #use the indexed field
-    user = User.find_or_initialize_by(display_name: login)
+    user = User.find_or_initialize_by(login: login)
```

Comment:
If we're converting zoo home login fields to a restricted char set then these finds won't work by default. Suggest we add a sanitized login method on the User model to handle the conversion.